contracts/modules/RecoveryManager.sol

```diff
@@ -80,13 +117,18 @@ contract RecoveryManager is BaseModule, RelayerModule {
         ModuleRegistry _registry,
         GuardianStorage _guardianStorage,
         uint256 _recoveryPeriod,
-        uint256 _lockPeriod
+        uint256 _lockPeriod,
+        uint256 _securityPeriod,
+        uint256 _securityWindow
     )
         BaseModule(_registry, _guardianStorage, NAME)
         public
     {
+        guardianStorage = _guardianStorage;
         recoveryPeriod = _recoveryPeriod;
         lockPeriod = _lockPeriod;
+        securityPeriod = _securityPeriod;
```

Comment:
I think we should add a condition for `recoveryPeriod` `securityPeriod` and `securityWindow` in the constructor as these 3 parameters must be related for the wallet to be secure. In particular they must satisfy `recoveryPeriod >= securityPeriod + securityWindow`.